kivy/uix/widget.py

```diff
@@ -250,6 +250,9 @@ def on_touch_down(self, touch, after=False):
 from kivy.weakproxy import WeakProxy
 from functools import partial
 from itertools import islice
+from io import BytesIO
+from PIL import Image
```

Comment:
Importing PIL in Widget is a absolute no-go!

Reply:
If you know another way around it, I would like to know.